src/server/scripts/Northrend/Naxxramas/naxxramas.h

```diff
@@ -78,19 +82,20 @@ enum NXGOs
     GO_SAPPHIRON_BIRTH              = 181356,
     GO_KELTHUZAD_FLOOR              = 181444,
     GO_KELTHUZAD_GATE               = 181228,
+    GO_KELTHUZAD_PORTAL_1           = 181402,
+    GO_KELTHUZAD_PORTAL_2           = 181403,
+    GO_KELTHUZAD_PORTAL_3           = 181404,
+    GO_KELTHUZAD_PORTAL_4           = 181405,
 
-    GO_DEATHKNIGHT_WING             = 181577, //Loatheb portal
-    GO_THADDIUS_PORTAL              = 181576, //Thadius portal
-    GO_MAEXXNA_PORTAL               = 181575, //Maexxna portal
-    GO_HORSEMAN_PORTAL              = 181578, //Four Horseman portal
+    GO_DEATHKNIGHT_WING             = 181577,
+    GO_THADDIUS_PORTAL              = 181576,
+    GO_MAEXXNA_PORTAL               = 181575,
+    GO_HORSEMAN_PORTAL              = 181578,
```

Comment:
why did you remove the comments?

Reply:
Redundant commentaries, it was already defined as "GO_BOSS_PORTAL" so no point in having a commentary saying // boss_portal